### PR TITLE
put selection handle outside selection

### PR DESCRIPTION
**Pull request by Doublonmousse**
image

### CODE REVIEW

**Comment by Doublonmousse** [comment]
I've updated the minimum size (for now it's set at `2.0 / zoom_level`).
I don't see any of the issues I had with me setting it to 0.01 so that's good.

Having smaller minimum selection size does make #1017 more prominent though.
I feel like changing the stroke width calculation so that the following are true could help a little though (maybe a pattern matching could suffice)
- scale =(1,r) => scale_stroke = r
- scale =(r,1) => scale_stroke = r
- scale = (x,y) => scale_stroke = (x+y)/2

At least it would remove the issue for resize that are done on a single direction, leaving only the case of a series of resizes that are neither uniform nor done in a single direction without having to calculate the stroke width with respect to the initial size of the selection

File: crates/rnote-engine/src/pens/selector/penevents.rs [hunk 1]
```diff
@@ -321,8 +321,7 @@ impl Selector {
                             let offset_mean = offset_to_start.mean();
                             offset_to_start = start_extents * (offset_mean / start_mean);
                         }
-                        let min_extents = (Self::RESIZE_NODE_SIZE
-                            + na::Vector2::<f64>::from_element(Self::ROTATE_NODE_DIAMETER))
+                        let min_extents = (na::Vector2::<f64>::from_element(2.0f64))
```
Comment (flxzt) [inline crates/rnote-engine/src/pens/selector/penevents.rs:324]: unneeded parentheses here

**Comment by flxzt** [comment]
That's a very interesting idea, let's discuss it further in #1017

File: crates/rnote-engine/src/pens/selector/penevents.rs (hunk 1 above)
Comment (Doublonmousse) [reply crates/rnote-engine/src/pens/selector/penevents.rs]: fixed

**Comment by flxzt** [comment]
Thank you! :)

**Pull request merged**